nexus/db-model/src/deployment.rs

```diff
@@ -27,7 +30,7 @@ use nexus_types::deployment::BlueprintZoneDisposition;
 use nexus_types::deployment::BlueprintZonesConfig;
 use nexus_types::deployment::CockroachDbPreserveDowngrade;
 use nexus_types::deployment::{
-    blueprint_zone_type, BlueprintPhysicalDisksConfig,
+    blueprint_zone_type, BlueprintPhysicalDisksConfig, ClickhouseClusterConfig,
```

Comment:
The `ClickhouseClusterConfig` must be generated from the 3 tables. I plan to do that inside the db-query for blueprints.